weather_dl/download_pipeline/pipeline.py

```diff
@@ -163,6 +163,9 @@ def run(argv: t.List[str], save_main_session: bool = True) -> PipelineArgs:
             config.get('parameters', {}).get('dataset', "")
         )
 
+    logger.warning(f'By using {client_name} datasets, '
+                   f'users agree to the terms and conditions specified in {CLIENTS[client_name](config).license_url}')
```

Comment:
Let's instantiate one client only in this method. So, how about we pull out the client in line 162 into a single variable that's used to count the number of requests and to log this warning?

Reply:
Said code-changes implemented.